src/server/data-access/error/EntityNotFound.ts

```diff
@@ -0,0 +1 @@
+export default class EntityNotFound extends Error {}
```

Comment:
Do we want that exception on the stack, like in src/common/exceptions/LogicException.ts ?

Reply:
You mean generally creating only exception types that extend (our) generic exception types? I could get behind this for the sake of clustering them (less so for the stack trace part).
However I did not really have them on my radar so far and how their inheritance tree looks like. Having taken a brief look I'd shy away from building on top of them as the first class I opened shows, in my eyes, a problem: `RuntimeException` should not extend `LogicException` per my understanding. [Compare](http://php.net/manual/en/spl.exceptions.php#spl.exceptions.tree)